sim/validator.py

```diff
@@ -114,6 +116,40 @@ def pipeline(
         shutil.rmtree(btb_seq_backup_path)
         shutil.rmtree(results_path)
 
+def sequence_and_benchmark(btb_seq_path, genomes_path, reads_path, output_path, light_mode):
+    # Load
+    genomes = genome.from_directory(genomes_path)
+    read_pairs = reads.from_directory(reads_path)
+
+    if not utils.names_consistent(genomes, read_pairs):
+        raise Exception("Genomes and read names not consistent")
+
+    # Initialise
+    results_path = os.path.join(output_path, 'sequenced')
+    stats_path = os.path.join(output_path, 'stats')
+    btb_seq_backup_path = os.path.join(output_path, 'btb-seq')
+
+    os.makedirs(stats_path, exist_ok=False)
+
+    shutil.copytree(btb_seq_path, btb_seq_backup_path, symlinks=True)
+    
+    # Sequence
+    sequenced_samples = sequence(btb_seq_backup_path, reads_path, results_path)
```

Comment:
Feels like from here to 152 could by DRY'd with pipeline() i.e. have a new function something like "benchmark()". Maybe not essential or could be a TODO